osu.Game/Overlays/OverlayHeader.cs

```diff
@@ -5,14 +5,16 @@
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.UserInterface;
 using osu.Game.Graphics.UserInterface;
 using osuTK.Graphics;
 
 namespace osu.Game.Overlays
 {
-    public abstract class OverlayHeader : Container
+    public abstract class OverlayHeader<TModel> : Container
```

Comment:
what on earth is `TModel`? This needs documentation and potentially better naming.

Reply:
on top of that, nothing uses this class. is there a point to splitting it out as you have done?

Reply:
There's a `BreadcrumbControlOverlayHeader` and `TabControlOverlayHeader`, which use this class. (If it what you mean)
I'll add xmldoc